app/controllers/validators_controller.rb

```diff
@@ -45,8 +45,11 @@ def show
     time_from = Time.now - 24.hours
     time_to = Time.now
 
+    @validator.name = @validator.name&.gsub(" ", "-space-")
```

Comment:
Why are we doing this?
Also, why are we using this method instead of updating API controller?